chromadb/types.py

```diff
@@ -168,9 +168,7 @@ class Segment(TypedDict):
     id: UUID
     type: NamespacedName
     scope: SegmentScope
-    # If a segment has a collection, it implies that this segment implements the full
-    # collection and can be used to service queries (for it's given scope.)
-    collection: Optional[UUID]
```

Comment:
discussed with @HammadB, we agreed that all segments should have collection IDs and we can express segment relationships via a different field